.circleci/integration-tests/master_dag.py

```diff
@@ -242,6 +242,18 @@ def prepare_dag_dependency(task_info, execution_time):
         provide_context=True,
     )
 
+    end_test = DummyOperator(
+        task_id="end_test",
+        trigger_rule="all_success",
+    )
+
+    aws_nuke_task_info = [
+        {"aws_nuke_dag": "example_aws_nuke"},
```

Comment:
The example_aws_nuke has its schedule set to 8:30 PM, that might need to be corrected. WDYT?

Reply:
shall I set to None?

Reply:
yeah please set it to None

Reply:
ok, set this the schedule None for AWS nuke DAG